stdlib/public/core/Arrays.swift.gyb

```diff
@@ -1412,6 +1416,7 @@ extension Array {
   ///
   /// - Complexity: O(`self.count`) if the array is bridged,
   ///   otherwise O(1).
+  @discardableResult
   public mutating func popLast() -> Element? {
```

Comment:
Any idea which is preferable for the consistent solution? They seem to be so close to `remove` and `removeLast` that they should have the same "discardability", meaning that all of the `pop*` functions should have `@discardableResult`.